src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,115 @@
+# Tuning Component Settings
+
+Helm Charts are a set of Kubernetes manifests that reflect best practices for deploying an
+application or service. Helm is heavily influenced by [Homebrew](http://brew.sh/), including the
+[formula model](https://github.com/Homebrew/homebrew-core). A Helm Chart is to Helm as a Formula
+is to Homebrew.
+
+When you run `helmc fetch deis/workflow-beta4`, you can customize the chart with
+`helmc edit workflow-beta4`. To customize the respective component, edit
+`manifests/deis-<component>-rc.yaml` and modify the `env` section of the component to tune these
+settings.
+
```

Comment:
Suggest we reword to:

> After you run `helmc fetch deis/workflow-2.0.0`, you can customize the chart using `helmc edit workflow-2.0.0` before using `helmc generate workflow-2.0.0` and `helmc install workflow-2.0.0` to complete the installation.

I think this gives a better feel for the overall process. The rest of the wording in that paragraph is fine.